documentation/0.13.0/html/proxy-quick-start/index.markdown

```diff
@@ -152,3 +152,117 @@ kaf -b $KROXYLICIOUS_BOOTSTRAP consume my_topic
 [//]: # (====================================================================)
 [//]: # (END - Use Section Examples Tabbed Card)
 [//]: # (====================================================================)
+
+# Running Kroxylicious via Docker Compose
+
+## Prerequisites
+
+Make sure to have either [Docker Compose](https://docs.docker.com/compose/) or [Podman Compose](https://docs.podman.io/en/stable/markdown/podman-compose.1.html) installed.
+
+## Step 1: Create the Compose file
+
+Create a file named _docker-compose.yaml_ with the following contents:
+
+```yaml
+services:
+  # from https://github.com/apache/kafka/blob/trunk/docker/examples/docker-compose-files/single-node/plaintext/docker-compose.yml
+  kafka:
+    image: apache/kafka:4.0.0
+    ports:
+      - '9092:9092'
```

Comment:
question: Do we need to expose any port here?

The (local) clients will use Kroxylicious to connect to the cluster, no?

Reply:
Strictly speaking, it's not needed. I still like it, as it _allows_ local clients to connect directly, when needed.